sdcm/utils/sstable/load_utils.py

```diff
@@ -91,7 +91,7 @@ def run_refresh(node, test_data: namedtuple) -> Iterable[str]:
         # So we need to validate that resharded files are placed in the "upload" folder before moving.
         # Find the compaction output that reported about the resharding
 
-        system_log_follower = node.follow_system_log(patterns=[r'Resharded.*\[/'])
+        system_log_follower = node.follow_system_log(patterns=[r'Resharded.*'])
```

Comment:
follow_system_log is using re.search on these patterns, so `.*` is not needed.
Also I wonder if we could be more precise here and have something like `(\] Resharded [0-9]+ sstables to \[|\] database - Resharded )`

Reply:
Look at new docstring lines. The reason of this PR is that lines structure has changed. Your proposal considers only old variant, which will fail on the latest Scylla master. Current version of PR, even having ",*", will work for both.

Reply:
It has two parts in it, one matches old variant, other matches new one.
Problem with current variant is that it is unspecific and could easily appear somewhere that is not related to the resharding.

Reply:
Ok, I overlooked "|". 
So, both our variants work now. But, I don't see any real need to make it much stricter than it is now.
I don't have any idea where else word "Resharded" may be used except the "reshard" feature log messages. 
But having the pattern be stricter we may face such false non-findings more times.